packages/browser/src/integrations/breadcrumbs.ts

```diff
@@ -170,7 +170,7 @@ export class Breadcrumbs implements Integration {
     const client = getCurrentHub().getClient<BrowserClient>();
     const dsn = client && client.getDsn();
     if (this._options.sentry && dsn) {
-      const filterUrl = new API(dsn).getStoreEndpoint();
+      const filterUrl = new API(dsn).getBaseApiEndpoint();
```

Comment:
This is used to decide whether a breadcrumb is internal or not -- now that there are multiple endpoints, the filter needs to consider a broader pattern `.../api/`.

Is it okay to match `.../api/*` or do we need to include the `projectId` in the filter `.../api/PROJ_ID/*`?
Note that `.../api/*` will also match the endpoint used for the "Report Dialog", which was not matched previously.

Reply:
I think it should be fine to filter on broader pattern